test/Mono.Linker.Tests.Cases/DataFlow/ArrayDataFlow.cs

```diff
@@ -227,7 +227,7 @@ static void TestArrayResetStoreUnknownIndex (int i = 0)
 		// https://github.com/dotnet/linker/issues/2680 - analyzer doesn't reset array in this case
 		[ExpectedWarning ("IL2062", nameof (DataFlowTypeExtensions.RequiresPublicFields), ProducedBy = ProducedBy.Trimmer)]
 		// https://github.com/dotnet/linker/issues/2632 - Ref params don't reset or track.
-		// [ExpectedWarning ("IL2062", nameof (DataFlowTypeExtensions.RequiresPublicMethods))]
```

Comment:
Should we now delete the comment with the link to the issue (if this specific case is already fixed)